### PR TITLE
Removed static user, changed cart for admin

### PR DESCRIPTION
**Pull request by Nokafan**

### CODE REVIEW

File: src/main/java/com/internet/shop/controllers/cart/admin/DeleteProductFromCartAdminController.java [hunk 1]
```diff
@@ -20,8 +19,11 @@
...
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        Long id = Long.valueOf(req.getParameter("id"));
-        cartService.deleteProduct(cartService.getByUserId(USER_ID), productService.get(id));
-        resp.sendRedirect(req.getContextPath() + "/carts/edit");
+        Long productId = Long.valueOf(req.getParameter("productId"));
+        Long cartId = Long.valueOf(req.getParameter("cartId"));
+        cartService.deleteProduct(cartService.get(cartId),
+                productService.get(productId));
+        req.setAttribute("cart", cartService.get(cartId));
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/controllers/cart/admin/DeleteProductFromCartAdminController.java]: let's save cartService.get() to local variable

File: src/main/java/com/internet/shop/controllers/cart/admin/GetUsersShoppingCartsAdminController.java [hunk 2]
```diff
@@ -8,8 +8,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-public class GetUserShoppingCartAdminController extends HttpServlet {
-    private static final Long USER_ID = 1L;
+public class GetUsersShoppingCartsAdminController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/controllers/cart/admin/GetUsersShoppingCartsAdminController.java]:
```suggestion
public class GetAllShoppingCartsAdminController extends HttpServlet {
```

File: src/main/java/com/internet/shop/controllers/user/LogoutController.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.controllers.user;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/controllers/user/LogoutController.java]: this should be in the next PR

File: src/main/webapp/web.xml [hunk 4]
```diff
@@ -225,4 +256,6 @@
         <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+
+
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
```

File: src/main/java/com/internet/shop/controllers/cart/admin/DeleteProductFromCartAdminController.java (hunk 1 above)
Comment (Nokafan) [reply src/main/java/com/internet/shop/controllers/cart/admin/DeleteProductFromCartAdminController.java]: Fixed to new variable userCart

File: src/main/java/com/internet/shop/controllers/cart/admin/GetUsersShoppingCartsAdminController.java (hunk 2 above)
Comment (Nokafan) [reply src/main/java/com/internet/shop/controllers/cart/admin/GetUsersShoppingCartsAdminController.java]: Fixed.

File: src/main/webapp/web.xml (hunk 4 above)
Comment (Nokafan) [reply src/main/webapp/web.xml]: Fixed

File: src/main/java/com/internet/shop/controllers/user/LogoutController.java (hunk 3 above)
Comment (Nokafan) [reply src/main/java/com/internet/shop/controllers/user/LogoutController.java]: Did it fo the debagging? Can I leave it?

Comment (Nokafan) [reply src/main/java/com/internet/shop/controllers/user/LogoutController.java]: Deleted

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**